astropy/coordinates/tests/test_iau_fullstack.py

```diff
@@ -201,7 +203,12 @@ def test_future_altaz():
     # assured of being "fresh".  In this case getting times outside the range of the
     # table does not raise an exception.  Only if using IERS_B (which happens without
     # --remote-data, i.e. for all CI testing) do we expect another warning.
-    with pytest.warns(
+    if PYTEST_LT_8_0:
+        ctx1 = ctx2 = nullcontext()
+    else:
+        ctx1 = pytest.warns(erfa.core.ErfaWarning)
+        ctx2 = pytest.warns(AstropyWarning, match=r".*times are outside of range.*")
```

Comment:
Why all these raw strings (here and elsewhere)?
```suggestion
        ctx2 = pytest.warns(AstropyWarning, match=".*times are outside of range.*")
```

Reply:
Doesn't hurt. But I can remove if you want.